updatesh_d/makeit.sh

```diff
@@ -0,0 +1 @@
+ldc2 dlangorg/updatesh.d dlangorg/scriptutil.d
```

Comment:
ldc2 is less likely to be present than dmd. Anyhow this must go in the makefiles.